oresat_dxwifi/data/oresat_dxwifi.eds

```diff
@@ -4191,6 +4192,13 @@ ParameterName=Temperature
 ObjectType=0x07
 DataType=0x0008
 AccessType=ro
+DefaultValue=0
+
+[6002]
+ParameterName=New Variable
```

Comment:
ParameterName=Oresat Live Camera Resource Status

Reply:
File is no longer necessary.